src/app/blog/frost-threshold-signatures/page.mdx

```diff
@@ -0,0 +1,220 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import Image from 'next/image'
+
+export const post = {
+  draft: false,
+  author: 'Rüdiger Klaehn',
+  date: '2024-09-30',
+  title: 'Frost threshold signatures',
```

Comment:
```suggestion
  title: 'FROST threshold signatures',
```

I know, it's weird, but technically this is an abbreviation.
I'm not 100% convinced we should change it. Thoughts?

Reply:
I don't like the title anyway. It should be about why, not how. Your title with "drop the phone in the ocean" is better.